packages/dear-imgui/cpp/include/widget/text.h

```diff
@@ -23,8 +23,8 @@ class Text : public StyledWidget {
 
         void Patch(const json& widgetPatchDef, ReactImgui* view) override;
 
-        void Init() override {
-            Element::Init();
+        void Init(const json& elementDef) override {
+            Element::Init(elementDef);
```

Comment:
Instead of validating elementDef everywhere, I validate that it is an object and that it contains the id integer property one time only in ReactImgui::CreateElement. It's a calculated risk I am willing to take to boost performance a little bit.